source/_integrations/honeywell.markdown

```diff
@@ -48,7 +48,42 @@ The climate platform integrates Honeywell US-based thermostats into Home Assista
 
 All [climate services](/integrations/climate) are supported except set_swing_mode.
 
-There is a "known" issue related to setting the temperature from Home Assistant with some thermostats. If your instance receives errors when setting the temperature above or below some value, go to the Honeywell web page and set the temperature to the max or min for the mode you are using, then adjust from Home Assistant.
+Due to the instability of the Honeywell total connect system, service calls within automations should repeat until success similar to the following example:
+
+```yaml
```

Comment:
Please style this according to our YAML styleguide

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Reply:
The UI produces not our documentation style formatting (that is Home Assistant managed data in<->out from the YAML parser). The styling we look for is linked above.

Reply:
Thanks for the pointers.